<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Custom Item</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Number of Items</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>number_spinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>77</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" >
    <widget class="QSpinBox" name="number_spinBox" >
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Description</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>description_lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="QLineEdit" name="description_lineEdit" >
     <property name="maxLength" >
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Tooth (if applicable)</string>
     </property>
     <property name="buddy" >
      <cstring>tooth_lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="2" column="2" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>77</width>
       <height>68</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3" >
    <widget class="QLineEdit" name="tooth_lineEdit" >
     <property name="maxLength" >
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Fee</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>fee_doubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QDoubleSpinBox" name="fee_doubleSpinBox" >
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum" >
      <double>3000.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Patient Fee</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>ptFee_doubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QDoubleSpinBox" name="ptFee_doubleSpinBox" >
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum" >
      <double>3000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>number_spinBox</tabstop>
  <tabstop>description_lineEdit</tabstop>
  <tabstop>tooth_lineEdit</tabstop>
  <tabstop>fee_doubleSpinBox</tabstop>
  <tabstop>ptFee_doubleSpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
